Sources/Instrumentation/SignPostIntegration/SignPostIntegration.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
let's remove the year everywhere =)

Reply:
I will replace with the format I found in opentelemetry-java:
```
/*
 * Copyright The OpenTelemetry Authors
 * SPDX-License-Identifier: Apache-2.0
 */
```
If approved I will create another PR changing to this header in all project files